tests/test_docking.py

```diff
@@ -3,18 +3,32 @@
 """
 import tempfile
 import unittest
+from importlib import resources
 from pathlib import Path
 
+from rdkit import Chem
+
 from docking import dock_and_equilibrate, pose_dock, relative_docking, rigorous_work
 from timemachine.testsystems.relative import hif2a_ligand_pair
 
 
+def get_hif2a_ligands_as_sdf_file(num_mols: int) -> tempfile.NamedTemporaryFile:
+    mols = []
+    with resources.path("timemachine.testsystems.data", "ligands_40.sdf") as path_to_ligand:
+        suppl = Chem.SDMolSupplier(str(path_to_ligand), removeHs=False)
+        for _ in range(num_mols):
+            mols.append(next(suppl))
+    temp_sdf = tempfile.NamedTemporaryFile(suffix=".sdf")
+    with Chem.SDWriter(temp_sdf.name) as writer:
+        for mol in mols:
+            writer.write(mol)
+    return temp_sdf
+
+
 class TestDocking(unittest.TestCase):
     def test_pose_dock(self):
         """Tests basic functionality of pose_dock"""
-        guests_sdfile = str(
-            Path(__file__).resolve().parent.parent.joinpath("tests", "data", "ligands_40__first-two-ligs.sdf")
-        )
+        temp_sdf = get_hif2a_ligands_as_sdf_file(1)
```

Comment:
nit: slightly more readable?
```suggestion
        temp_sdf = get_hif2a_ligands_as_sdf_file(num_mols=1)
```